lms/djangoapps/teams/static/teams/js/views/edit_team.js

```diff
@@ -0,0 +1,174 @@
+;(function (define) {
+'use strict';
+
+define(['backbone',
+        'underscore',
+        'gettext',
+        'js/views/fields',
+        'teams/js/models/team',
+        'text!teams/templates/edit-team.underscore'],
+       function (Backbone, _, gettext, FieldViews, TeamModel, edit_team_template) {
+           return Backbone.View.extend({
+
+               maxTeamNameLength: 255,
+               maxTeamDescriptionLength: 300,
+
+               events: {
+                   "click .action-primary": "createTeam",
+                   "click .action-cancel": "cancelTeam"
+               },
+
+               initialize: function(options) {
+                   this.courseId = options.teamParams.courseId;
+                   this.teamsUrl = options.teamParams.teamsUrl;
+                   this.topicId = options.teamParams.topicId;
+                   this.languages = options.teamParams.languages;
+                   this.countries = options.teamParams.countries;
+
+                   _.bindAll(this, "cancelTeam", "createTeam");
+
+                   this.teamNameField = new FieldViews.TextFieldView({
+                       model: new TeamModel(),
+                       title: gettext("Team Name (Required) *"),
+                       valueAttribute: 'name',
+                       bindEvents: false,
+                       helpMessage: gettext("A name that identifies your team (maximum 255 characters).")
```

Comment:
@frrrances We'd like to add something to let students know that team metadata can't be changed after the team has been created. 
We have no place where we can cover all fields, so I'm suggesting adding a sentence after the existing text under each field. But is the help text getting too long?
Add: "You cannot change the team name after you create the team."